lighthouse-core/audits/load-fast-enough-for-pwa.js

```diff
@@ -78,12 +82,18 @@ class LoadFastEnough4Pwa extends Audit {
           value: {areLatenciesAll3G, allRequestLatencies, isFast, timeToFirstInteractive}
         };
 
+        const details = Audit.makeV2TableDetails([
+          {key: 'url', itemType: 'url', text: 'URL'},
+          {key: 'latency', itemType: 'text', text: 'Latency (ms)'},
```

Comment:
Is a cached column worth showing? In theory, this table doesn't contain any requests that come from the cache.